t/12enumq.t

```diff
@@ -0,0 +1,38 @@
+=pod
+
+=encoding utf-8
+
+=head1 PURPOSE
+
+Tests for C<Enum> for Type::Tiny > 1.010003 with EnumQ encoding
+
+=head1 AUTHOR
+
+Toby Inkster E<lt>tobyink@cpan.orgE<gt>.
+
+=head1 COPYRIGHT AND LICENCE
+
+This software is copyright (c) 2014 by Toby Inkster.
+
+This is free software; you can redistribute it and/or modify it under
+the same terms as the Perl 5 programming language system itself.
+
+=cut
+
+use strict;
+use warnings;
+use Test::More tests => 10;
+
+use_ok('Type::Tiny::XS');
+
+my $check = Type::Tiny::XS::get_coderef_for('EnumQ["a b", "c, d", "-\""]');
```

Comment:
This should be `Enum`, not `EnumQ`, right?

Reply:
Yep. I have now figured out distinkt-dist :).  Tests are passing now.